CodeStarterCamp_Week1/main.swift

```diff
@@ -8,5 +8,37 @@
 
 import Foundation
 
-print("Hello, World!")
+//아이스크림 부분을 그리는 함수
+/*
+func drawIceCreamPart(){
+    for _ in 1...8{
+        print("***********")
+    }
+}
+ */
 
+drawIceCreamPart()
+
+//이렇게 하면 줄바꿈을 두번씩 함. -> 한번씩만 할 수 있는 방법은 없을까?
+
+func drawIceCreamPart(){
```

Comment:
함수명을 알맞게 잘 지어주셨군요! 덕분에 주석이 없더라도 어떤 기능을 하는 함수인지 직관적으로 알 수 있네요~!👍
다만, [API Design Guidelines](https://www.swift.org/documentation/api-design-guidelines/)에 따르면 중괄호는 한칸 띄워서 작성합니다! 가이드라인을 잘 읽어보세요!
```suggestion
func drawIceCreamPart() {
```